parsl/dataflow/dflow.py

```diff
@@ -1015,6 +1013,10 @@ def load(cls, config=None):
 
         return cls._dfk
 
+    @classmethod
```

Comment:
Should we add a shortcut `wait_for_current_tasks = DataFlowKernelLoader.wait_for_current_tasks` in `__init__.py`?

Reply:
Done